packages/packages_by_dep_dag.txt

```diff
@@ -1,21 +1,27 @@
 util
-schema
+
 config
 files
 job_metrics
 navigation
 objectstore
+schema
```

Comment:
Conceptually I preferred this package higher to be more explicit about it not having a lot of dependencies. But you've broken it up here into like levels and alphabetized them... I guess this is okay. The spaces are certainly helpful and appreciated. Thank you

Reply:
Yes, for me at least, it makes it easier to visualise that the packages in this group form a layer that only depend on the above layers, in this case `util`.